Support/Python/unitybuild/main.py

```diff
@@ -252,6 +252,14 @@ def iter_editors_and_versions():  # pylint: disable=too-many-branches
             editor_data_dir = os.path.join(editor_dir, "Contents")
             if os.path.exists(editor_dir):
                 yield (exe, get_editor_unity_version(editor_dir, editor_data_dir))
+    elif sys.platform.startswith('linux'):
+        editor_dir = os.path.expanduser('~/Unity/Hub/Editor/2019.4.32f1')
```

Comment:
can you loosen this a bit to accept any version, at least within the 2019.* range? Compare, for example, lines 248-254 for Mac

Reply:
will fix this to allow for the 2019.* range